ansible/group_vars/release.yml

```diff
@@ -1,6 +1,5 @@
 jenkins_url: "https://ci-release.nodejs.org"
-# intentionally fetching `slave.jar` from ci.nodejs.org to avoid auth problems
-jenkins_worker_jar: "https://ci.nodejs.org/jnlpJars/slave.jar"
+jenkins_worker_jar: "{{ jenkins_url }}/jnlpJars/slave.jar"
```

Comment:
Aside: weren't these jars going to be renamed since Jenkins had updated naming V2?

Reply:
Yes, we can use `agent.jar` now. I'll leave that change for another PR.